website/site/content/docs/authentication-backends.md

```diff
@@ -74,28 +73,78 @@ can use one of the community-maintained projects below. Feel free to [submit a p
 
 Check each project's documentation for instructions on how to configure it.
 
-## Bitbucket and GitLab Support
+## GitLab Backend
+
+For repositories stored on GitLab, the `gitlab` backend allows CMS users to log in directly with their GitLab account. Note that the user's GitLab account must have push access to your content repository for this to work.
+
+Unlike GitHub, the GitLab API allows for two types of OAuth2 flows: [Web Application Flow](https://docs.gitlab.com/ce/api/oauth2.html#web-application-flow), which works much like the GitHub OAuth flow described above, and [Implicit Grant](https://docs.gitlab.com/ce/api/oauth2.html#implicit-grant), which operates _without_ the need for an authentication server.
+
+### Web Application Flow with Netlify
+
+When using GitLab's Web Application Flow for authentication, you can use Netlify to handle the server-side authentication requests.
+
+To enable it:
+
+1. Follow the [GitLab docs](https://docs.gitlab.com/ee/integration/oauth_provider.html#adding-an-application-through-the-profile) to add your Netlify CMS instance as an OAuth application. For the **Redirect URI**, enter `https://api.netlify.com/auth/done`, and check the box for `api` scope.
+2. Follow the [Netlify
+   docs](https://www.netlify.com/docs/authentication-providers/#using-an-authentication-provider) to add your new GitLab Application ID and Secret to your Netlify site dashboard. 
+2. In your repository, add the following lines to your `config.yml` file:
```

Comment:
Very good point, especially since Jekyll uses the filename of `config.yml` as well. Maybe
> add the following lines to your ***Netlify CMS*** `config.yml` file

Reply:
Updated throughout the auth doc, then went ahead and did it for all the docs. :)